hack/arktos-up-scale-out-poc.sh

```diff
@@ -14,13 +14,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [[ -z "${SCALE_OUT_PROXY_ENDPOINT}" ]]; then
-  echo ERROR: Please set SCALE_OUT_PROXY_ENDPOINT. For example: SCALE_OUT_PROXY_ENDPOINT=\"http://192.168.0.120:8888\"
-  exit 1
-fi
-
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
+if [[ "${INSTALL_SCALE_OUT_PROXY}" == "true" ]]; then
+  if [ "${IS_RESOURCE_PARTITION}" == "true" ]; then
+    RESOURCE_PARTITION_IP= TENANT_PARTITION_IP=${SECOND_PARTITION_IP} ${KUBE_ROOT}/hack/scale_out_poc/setup_nginx_proxy.sh
```

Comment:
missing something here after the first "=" ?

Reply:
Thanks for catching it. Fixed.